base/util.go

```diff
@@ -1034,3 +1038,22 @@ func ReplaceAll(s, chars, new string) string {
 	}
 	return s
 }
+// Make a deep copy from src into dst.
+// Copied from https://github.com/getlantern/deepcopy, commit 7f45deb8130a0acc553242eb0e009e3f6f3d9ce3 (Apache 2 licensed)
```

Comment:
I'm not convinced we want to add DeepCopy as a generic utility function, given it's limitations:
- it only copies exported fields
- poor performance

This seems like it might make more sense as a test-only helper function on DbConfig.

At minimum I'd prefer to keep this in util_testing.go, to help ensure it's not inadvertently used in production code.

Reply:
How about renaming to `DeepCopyInefficient()` to make it impossible to accidentally using this in a critical section?  (in addition to moving to `util_testing.go`)